pandas/tests/frame/test_analytics.py

```diff
@@ -875,11 +875,6 @@ def test_mean_datetimelike(self):
         expected = pd.Series({"A": 1.0, "C": df.loc[1, "C"]})
         tm.assert_series_equal(result, expected)
 
-    @pytest.mark.xfail(
```

Comment:
I guess you should technically have a whatsnew note as this 'bug' is fixed (do in followon)